downstream/modules/platform/proc-update-aap-rpm.adoc

```diff
@@ -0,0 +1,131 @@
+[id="proc-update-aap-rpm"]
+
+== Update planning
+
+Before you begin the update process, review the following considerations to plan and prepare your {PlatformNameShort} deployment:
+
+* Even if you have a valid license from a previous version, you must provide your credentials or a subscription manifest upon upgrading to the latest version of {PlatformNameShort}. For more information, see link:docs.redhat.com/en/documentation/{PlatformName}/2.5/html/access_management_and_authentication/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching your {PlatformName} subscription].
```

Comment:
alternately formatted link: 
link:{URLCentralAuth}/assembly-gateway-licensing#proc-attaching-subscriptions[Attaching your {PlatformName} subscription]

Reply:
File no longer used. It has been removed.